website/content/docs/east-west/network-area.mdx

```diff
@@ -106,106 +362,184 @@ world
+Once the command is execute from one of the datacenters, the federation is already removed.
 
 <CodeBlockConfig heading="server.dc1">
 
 ```shell-session
-$ consul operator area members
-Area                                  Node        Address           Status  Build      Protocol  DC   RTT
-beb39435-43e8-5979-7c11-b5e011e04f51  server.dc1  10.20.10.11:8300  alive   1.6.2+ent  2         dc1  0s
-beb39435-43e8-5979-7c11-b5e011e04f51  server.dc2  10.20.10.12:8300  left    1.6.2+ent  2         dc2  4.391502ms
+$ curl -s http://127.0.0.1:8500/v1/operator/area/1c8cd5e6-562c-e0de-e369-a13c6205ffe8/members
 ```
 
 </CodeBlockConfig>
 
-### Functionality compatibility
-
-Network areas are the suggested approach when your architecture layout does not permit a full mesh connectivity between all servers across datacenters. Due to the same connectivity constraints, some of the latest Consul functionalities do not have full compatibility with network areas.
+The server is now shown as `left` from the other datacenters.
+
+<CodeBlockConfig hideClipboard highlight="11">
+
+```json
+[
+  {
+    "ID": "55e61b05-7648-d0d7-e868-6e56e878e371",
+    "Name": "consul-server-1.dc2",
+    "Addr": "172.18.0.3",
+    "Port": 8300,
+    "Datacenter": "dc2",
+    "Role": "server",
+    "Build": "1.20.5+ent",
+    "Protocol": 2,
+    "Status": "left",
+    "RTT": 2962233
+  },
+  {
+    "ID": "6e4910c1-0259-f737-1ae9-8807b2816d0b",
+    "Name": "consul-server-0.dc1",
+    "Addr": "172.18.0.5",
+    "Port": 8300,
+    "Datacenter": "dc1",
+    "Role": "server",
+    "Build": "1.20.5+ent",
+    "Protocol": 2,
+    "Status": "alive",
+    "RTT": 0
+  }
+]
+```
 
-In case you are trying to setup ACL replication using the [1.4 ACL system](/consul/docs/secure/acl) or enabling Consul service mesh with CA replication, we recommend you to use WAN gossip federation to leverage all Consul's latest functionalities.
+</CodeBlockConfig>
 
-Future version of Consul will gradually integrate all functionalities into network areas.
+</Tab>
+</Tabs>
 
-## Next steps
+## Data replication
 
-In this tutorial, you setup advanced federation using network areas. You then learned how to route RPC commands and use the DNS interface with multiple datacenters.
\ No newline at end of file
+In general, data is not replicated between different Consul datacenters. When a request is made for a resource in another datacenter, the local Consul servers forward an RPC request to the remote Consul servers for that resource and return the results. If the remote datacenter is not available, then those resources will also not be available, but that won't otherwise affect the local datacenter. There are some special situations where a limited subset of data can be replicated, such as with Consul's built-in ACL replication capability, or external tools like [consul-replicate](https://github.com/hashicorp/consul-replicate/).
```

Comment:
```suggestion
In general, Consul does not replicate data between different Consul datacenters. When a request is made for a resource in another datacenter, the local Consul servers forward an RPC request to the remote Consul servers for that resource and return the results. If the remote datacenter is not available, those resources will also not be available. However, this will not affect the local datacenter. 

There are some special situations where a limited subset of data can be replicated, such as with Consul's built-in ACL replication capability, or external tools like [`consul-replicate`](https://github.com/hashicorp/consul-replicate/).
```